examples/pyplots/sigmoid_function.py

```diff
@@ -0,0 +1,35 @@
+"""
+============
+Infinite horizontal and vertical lines
+============
+A script that showes infinite horizontal and vertical lines together with a sigmoid function.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+t = np.linspace(-10, 10, 100)
+sig = 1 / (1 + np.exp(-t))
+
+plt.axhline(y=0, color="black", linestyle="--")
+plt.axhline(y=0.5, color="black", linestyle=":")
+plt.axhline(y=1.0, color="black", linestyle="--")
+plt.axvline(color="grey")
+plt.plot(t, sig, linewidth=2, label=r"$\sigma(t) = \frac{1}{1 + e^{-t}}$")
+plt.xlim(-10, 10)
+plt.xlabel("t")
+plt.legend(fontsize=14)
+plt.show()
+
+#############################################################################
+#
+# ------------
+#
+# References
+# """"""""""
+#
+# The use of the following functions, methods, classes and modules is shown
+# in this example:
+
+import matplotlib
+matplotlib.axes.Axes.axhline
```

Comment:
Please also add `matplotlib.pyplot.axhline` and `matplotlib.pyplot.axvline`. For such cases we reference both pyplot and Axes functions/method so that the example is linked to both.

Reply:
Clever